### PR TITLE
input: add base dark

### PR DESCRIPTION
**Pull request by oieusouamiguel**
## Context
Refactor `Input` component to use the Dark base.

## Checklist
- [x] Add Base Dark class to `former-kit-skin`
- [x] Add Base Dark props to Input component
- [x] Add Dark and Light bases in Former-Kit Storybook

## Linked Issues
- [x] Resolves pagarme/former-kit#193 in `former-kit` repository.

### CODE REVIEW

**Comment by alalbux** [comment]
could rebase with master? This pr is blocking https://github.com/pagarme/pilot/pull/922

File: src/styles/form-fields/input/index.css [hunk 1]
```diff
@@ -26,6 +26,60 @@
...
+    color: var(--form-item-dark-label-color);
+  }
+
+  & .container > textarea:disabled,
+  & .container > input:disabled {
+    border-color: var(--form-dark-border-color-disabled);
+    color: var(--form-dark-color-disabled);
+  }
+
+  & .disabled .secondaryText,
```
Comment (alalbux) [inline src/styles/form-fields/input/index.css]: the disabled is in container, so you need concat this values `&.disabled`

Comment (oieusouamiguel) [reply src/styles/form-fields/input/index.css]: thanks, @alalbux!

File: src/styles/form-fields/properties.css [hunk 2]
```diff
@@ -1,10 +1,25 @@
...
+
+  --form-item-dark-label-color: var(--color-light-silver-100);
   --form-item-label-color: var(--color-light-chromium-100);
+
   --form-item-secondary-font-size: 12px;
-  --form-item-color-disabled: var(--color-light-iron-50);
+
+  --form-dark-border-color-disabled: var(--color-light-grey-100);
+  --form-dark-color-disabled: var(--color-light-steel-100);
+  --form-light-color-disabled: var(--color-light-iron-50);
```
Comment (lucasjs) [inline src/styles/form-fields/properties.css]: Faltou uma variável para a borda light, que tem o valor `--color-light-iron-50)` e esta da linha 24 terá a var `--color-light-silver-100`.

Comment (oieusouamiguel) [reply src/styles/form-fields/properties.css]: obrigada!

File: src/styles/form-fields/input/index.css [hunk 3]
```diff
@@ -26,6 +26,75 @@
...
+    color: var(--form-item-light-color);
+  }
+
+  &.disabled .secondaryText {
+    color: var(--form-light-color-disabled);
+  }
+
+  & .container > textarea:disabled,
+  & .container > input:disabled {
+    border-color: var(--form-light-color-disabled);
```
Comment (rsmelo) [inline src/styles/form-fields/input/index.css]: essa borda deveria ser `--color-light-iron-50` segundo o zeplin

**Pull request merged**